src/js/pages/MyTrades/components/Trades.jsx

```diff
@@ -125,7 +124,7 @@ class Trades extends Component {
               const isBuyer = addressCompare(trade.buyer, this.props.address);
               const tradeStyle = getTradeStyle(trade, isBuyer);
 
-              return (<Card key={index} className="mb-3 shadow border-0 offer-card"
+              return (<Card key={index} className={classnames("mb-3", "shadow", "border-0", "offer-card", {"card-transparent": !this.props.active})}
```

Comment:
Just a tip. You can do:
```suggestion
              return (<Card key={index} className={classnames("mb-3 shadow border-0 offer-card", {"card-transparent": !this.props.active})}
```
Up to you